lms/templates/courseware/course_updates.html

```diff
@@ -16,7 +15,7 @@ <h2 class="date" id="msg-date-${index}">${update.get("date")}</h2>
         ></button>
       % endif
       <div class="toggle-visibility-element article-content ${'hidden' if index >= 1 else ''}" id="msg-content-${index}">
-        ${update.get("content")}
+       ${update.get("content")}
```

Comment:
Nit: this looks like an accidental change that lost one space of the indentation.